radicle/src/test/storage.rs

```diff
@@ -11,24 +12,40 @@ use crate::identity::IdentityError;
 
 pub use crate::storage::*;
 
+use self::refs::SignedRefs;
+
 #[derive(Clone, Debug)]
 pub struct MockStorage {
     pub path: PathBuf,
     pub inventory: HashMap<Id, Doc<Verified>>,
+    pub remotes: HashMap<Id, Vec<Remote>>,
```

Comment:
I think using `HashMap<PublicKey, SignedRefs>` is probably better than `Vec<Remote>` here. But I'm running out of time today to improve on that. I'm happy to refactor it in a new PR if needed.

Reply:
I've refactored this using `HashMap<>` instead, which I think is better, including the correctness: not to worry about the uniqueness of node Id of the remotes.